training_config/vision/vilbert_vgqa_pretrained.jsonnet

```diff
@@ -74,6 +74,9 @@ local vocabulary = if construct_vocab then {
   "data_loader": {
     "batch_size": gpu_batch_size,
     "shuffle": true,
+    "max_instances_in_memory": gpu_batch_size * 100,
```

Comment:
For my understanding, where is the 100 coming from?

Reply:
Totally arbitrary. In my benchmarking I found that `100 x batch_size` worked well.